contestants.json

```diff
@@ -89,4 +89,15 @@
             "twitter" : ""
         }
     ]
+},
+   {
+            "name" : "Divy Khare",
+            "imageurl" : "contestant/me.jpg",
+            "about" : "Learning OpenSource",
+            "facebook" : "https://www.facebook.com/divy.khare.3",
+            "github" : "https://github.com/Divy123",
+            "twitter" : ""
+        }
+    ]
```

Comment:
This was already there. No need to add again. Move your info above `]` bracket that exists there.